src/User32.Desktop/User32.cs

```diff
@@ -1488,7 +1508,7 @@ public static partial class User32
         ///     If a message is available, the return value is true.
         ///     <para>If no messages are available, the return value is false.</para>
         /// </returns>
-        [DllImport(nameof(User32), SetLastError = true)]
+        [DllImport(nameof(User32), SetLastError = true, CharSet = CharSet.Unicode)]
```

Comment:
Why set this to CharSet.Unicode ? Where is any string being marshaled? if you mean strings inside the MSG structure, it should be on the structure and not on the function, I think.